### PR TITLE
Feature/laura 7 24 15

### PR DESCRIPTION
**Pull request by veverkap**

### CODE REVIEW

File: _posts/2014-7-24-nice-to-meet-you [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: Nice to Meet You!
+tags: development
+author_github: laurawhalin
```
Comment (veverkap) [inline _posts/2014-7-24-nice-to-meet-you]: I added a few YAML configs

File: _posts/2014-7-24-nice-to-meet-you [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+Being a mother of a young child, I knew I couldn't fit extra time into my day to
+learn how to code. Then one of those "right place at the right time" moments
+happened and I heard a news story on the radio about a local coding school. It
+sounded intriguing because I could learn how to code quickly, thoroughly, and if
+developers were really as in-demand as everyone was saying, I could find a job
+without much trouble. And I would be able to balance my life at home. I decided
+to take a risk.
+
+I looked into the local school options, applied, and was accepted to The
+Turing School of Software and Design in Denver within a short amount of time. I
```
Comment (veverkap) [inline _posts/2014-7-24-nice-to-meet-you]: Can we add a link here?

[Turing School of Software and Design](http://turing.io)

**Comment by laurawhalin** [comment]
Well I learned a lot with this pull request! I think everything is good to go for Wednesday, unless merging master was a no-no. :smile:

**Comment by veverkap** [comment]
Hah.  It's confusing because we don't do it this way on the other apps.

Branch from develop.  
Merge develop into your branch.  
Make pull request from your branch INTO develop.
Make release branch from develop.
Make pull request from release branch INTO master.

**Comment by veverkap** [comment]
See [this](https://www.atlassian.com/git/images/tutorials/collaborating/comparing-workflows/gitflow-workflow/04.svg)

**Pull request merged**